rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -282,6 +282,7 @@ If your organization does any automated registration, then you'll need to authen
    Flytectl's `config.yaml <https://docs.flyte.org/projects/flytectl/en/stable/#configure>`_ can be
    configured to use either PKCE (`Proof key for code exchange <https://datatracker.ietf.org/doc/html/rfc7636>`_)
    or Client Credentials (`Client Credentials <https://datatracker.ietf.org/doc/html/rfc6749#section-4.4>`_) flows.
+   If you are using the internal auth server, you can obtain the client credential by ... (This is what we are trying to figure out)
```

Comment:
cc @EngHabu

by internal auth server you mean the one provided by Admin?

Reply:
or like, internal to your company?

Reply:
The one provided by admin. I assume even a company "internal" one would be configured as "external" in the flyteadmin config.

Reply:
True on both accounts... 
So admin is deployed with a default [config here](https://github.com/flyteorg/flyteadmin/blob/12d6aa0a419ccec81b4c8289fd172e70a2ded525/auth/config/config.go#L86-L110). This can be overridden through the [config here](https://github.com/flyteorg/flyte/blob/1ede3750bfdeb307d1a81efd08674a68882bc873/kustomize/base/single_cluster/headless/config/admin/server.yaml#L26) by adding something like this:
```yaml
appAuth:
  selfAuthServer:
    staticClients:
      - newClient:
          id: newClient
          redirectUris:
            - http://myOtherApp...
          public: true
```